pkg/auxnetdevice/auxNetResourcePool.go

```diff
@@ -60,3 +64,8 @@ func (ap *auxNetResourcePool) GetDeviceSpecs(deviceIDs []string) []*pluginapi.De
 	}
 	return devSpecs
 }
+
+// GetPoolKind returns device kind for CDI spec
+func (ap *auxNetResourcePool) GetPoolKind() string {
+	return auxPoolType
```

Comment:
should be IMO:

net-<aux Type>  or net-aux-<type>

e.g net-sf or net-aux-sf

i prefer the former it will also be consistent with net-pci  for the other pool.

Reply:
done